app/pages/learn/01_tutorial/04_mastering-the-api/02_modern_io/index.md

```diff
@@ -0,0 +1,253 @@
+# Common I/O Tasks in Modern Java
+
+This article focuses on tasks that application programmers are likely to encounter, particularly in web applications, such as: 
+
+* Reading and writing text files
+* Reading text, images, JSON from the web
+* Visiting files in a directory
+* Reading a zip file
+* Creating a temporary file or directory
+
+The Java API supports many other tasks, which are explained in detail in the [Java I/O API tutorial](https://dev.java/learn/java-io/).
+
+Modern, at the time of this writing, means features that are out of preview in Java 21. In particular:
+
+* UTF-8 is the default for I/O since Java 18 ([JEP 400](https://openjdk.org/jeps/400))
```

Comment:
This is not mentioned anywhere else in this article.

Reply:
I added a sentence to explain it.